src/test/java/teammates/test/cases/action/AdminActivityLogPageActionTest.java

```diff
@@ -81,9 +81,19 @@ private void loadLogMessages() {
         }
     }
 
-    private void initVariable() {
-        formatterAdminTime = new SimpleDateFormat("dd/MM/yy");
-        formatterAdminTime.setTimeZone(TimeZone.getTimeZone(Const.SystemParams.ADMIN_TIME_ZONE));
+    private String formatAdminDate(Instant instant) {
+        return instant.atZone(Const.SystemParams.ADMIN_TIME_ZONE_ID).toLocalDate()
+                .format(DateTimeFormatter.ofPattern("dd/MM/yy"));
+    }
+
+    private String formatActivityLogTimeTruncated(Instant instant, ZoneId zoneId) {
```

Comment:
This replaces the deleted `SimpleDateFormat("dd-MM-yyyy HH")` that are used a lot below